grakn-test/test-snb/src/validate-snb/readwrite_grakn--ldbc_driver_config--db_validation.properties

```diff
@@ -0,0 +1,164 @@
+# -------------------------------------
```

Comment:
That file name!! `readwrite_grakn--ldbc_driver_config--db_validation.properties`